addons/website/controllers/main.py

```diff
@@ -383,24 +379,7 @@ def make_scss_custo(self, url, values):
                 word hook). If a key is already in the file's map, its value is
                 overridden.
         """
-        AssetsUtils = request.env['web_editor.assets']
-
-        custom_url = AssetsUtils.make_custom_asset_file_url(url, 'web.assets_common')
-        updatedFileContent = AssetsUtils.get_asset_content(custom_url) or AssetsUtils.get_asset_content(url)
-        updatedFileContent = updatedFileContent.decode('utf-8')
-        for name, value in values.items():
-            pattern = "'%s': %%s,\n" % name
-            regex = re.compile(pattern % ".+")
-            replacement = pattern % value
-            if regex.search(updatedFileContent):
-                updatedFileContent = re.sub(regex, replacement, updatedFileContent)
-            else:
-                updatedFileContent = re.sub(r'( *)(.*hook.*)', r'\1%s\1\2' % replacement, updatedFileContent)
-
-        # Bundle is 'assets_common' as this route is only meant to update
-        # variables scss files
-        AssetsUtils.save_asset(url, 'web.assets_common', updatedFileContent, 'scss')
-        return True
```

Comment:
Leave the `return True` here. Probably useless but at least we don't change behavior.